hoot-rnd/src/main/cpp/hoot/rnd/conflate/matching/ScoreMatchesDiff.cpp

```diff
@@ -114,10 +114,10 @@ void ScoreMatchesDiff::calculateDiff(const QString& input1, const QString& input
   std::shared_ptr<CountManualMatchesVisitor> manualMatchVisitor(
     new CountManualMatchesVisitor());
   map1->visitRo(*manualMatchVisitor);
-  const int numManualMatches1 = manualMatchVisitor->getStat();
+  double numManualMatches1 = manualMatchVisitor->getStat();
```

Comment:
Since matches are whole numbers and the `getStat()` function returns a `double` it needs to be cast to an integer.
```
const int numManualMatches1 = std::static_cast<int>(manualMatchVisitor->getStat());
```